docs/docsource/installation/install/install_xdm.rst

```diff
@@ -238,13 +235,11 @@ Deploy |product|
         version: |version|
         replicas: 1
         mongodb:
```

Comment:
the whole `mongodb` field is not correct: in the current model, we deploy mongodb beforehand, and add the params of the "external" mongodb in the CR

Reply:
I am not sure I understand, should I just remove the mongodb filed ? If not, then what should it be exactly ?

Reply:
I mean the `mongo` field configures Zenko for a specific mongo setup.

this doc was giving the whole kubedb config : changing the "provider" does not make the configuration correct, we need to update the doc with the current way to configure mongo in zenko cr (e.g. with external provider and with the other details)

you can check you artesca, you will see that the mongo field is not the same as all.

Reply:
I updated according to a real zenko in my platform
Needed to add a step to create the secret for mongodb credentials as it is required